llm_studio/python_configs/text_causal_language_modeling_config.py

```diff
@@ -174,8 +174,8 @@ def __post_init__(self):
         )
         self._possible_values["differential_learning_rate_layers"] = (
             possible_values.String(
-                values=("backbone", "embed"),
-                allow_custom=False,
+                values=("backbone", "embed", "head"),
```

Comment:
Why adding `head` to causal language modeling? Isn't it called `lm_head` for most models?

Reply:
It is just a substring match, so `head` works for most/all backbones.